vector/src/main/res/layout/fragment_location_sharing.xml

```diff
@@ -7,7 +7,9 @@
     <im.vector.app.features.location.MapTilerMapView
```

Comment:
not a blocker, something I noticed in the file layout - we could flatten the nested constraint layouts by using a `Group` with a background for the bottom bar

Reply:
Yes, Using a `Flow` is better. https://github.com/vector-im/element-android/pull/5062/commits/6107fb644d7e7810ff83da543700260c81fd5fce